packages/odyssey-storybook/src/components/odyssey-mui/Tag/Tag.stories.tsx

```diff
@@ -11,14 +11,14 @@
  */
 
 import { Story } from "@storybook/react";
-import { Chip, Stack } from "@okta/odyssey-react-mui";
+import { Tag, TagList } from "@okta/odyssey-react-mui";
 import { MuiThemeDecorator } from "../../../../.storybook/components";
 
 import TagMdx from "./Tag.mdx";
 
 export default {
   title: `MUI Components/Tag`,
-  component: Chip,
+  component: Tag,
   parameters: {
     docs: {
       page: TagMdx,
```

Comment:
We need to make sure we're typing the default export (which is `storybookMeta` as `Meta<TagProps>`); otherwise, we're not getting any benefit from TypeScript and will run into issues where our Storybook docs don't match the actual TS.

Reply:
Only problem with this is that the story params don't line up with the `TagProps`; for example, we allow the user to specify `label1`, `label2`, `label3` so they can show multiple tags in the canvas, but obviously those aren't in `TagProps`. Is there a way around this that still gets us all the benefits of TS?

Reply:
This seems to be a common theme in our stories — we have parameters that don't line up exactly with the component props.

Should we aim to keep those aligned? Or just not worry about using the types in the stories?